Service/Manager.php

```diff
@@ -337,6 +340,12 @@ public function commit(array $params = [])
                     break;
             }
 
+            if ($bulkResponse['errors']) {
+                throw new ClientErrorResponseException(
```

Comment:
I think we could pass also the whole bulk response to the exception message. No?